.github/workflows/publish-tanssi-relay-binary.yml

```diff
@@ -0,0 +1,133 @@
+name: Publish Dancelight Binary Draft
+
+# The code (like generate-release-body) will be taken from the tag version, not master
+on:
+  workflow_dispatch:
+    inputs:
+      from:
+        description: tag (ex. v0.8.3) to retrieve commit diff from
+        required: true
+      to:
+        description: tag (ex. v0.9.0) to generate release note and srtool runtimes from
+        required: true
+
+jobs:
+  ####### Building binaries #######
+  setup-scripts:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v4
+      - name: Upload tools
+        uses: actions/upload-artifact@v4
+        with:
+          name: original-tools
+          path: tools
+
+  build-binary:
+    needs: ["setup-scripts"]
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        cpu: ["", "skylake", "znver3"]
+    env:
+      RUSTFLAGS: "-C target-cpu=${{ matrix.cpu }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.to }}
+      - name: Setup Rust toolchain
+        run: rustup show
+      - name: Build Node
+        run: cargo build --profile=production --all
```

Comment:
cargo build --profile=production -p tanssi-relay

Reply:
Updated in https://github.com/moondance-labs/tanssi/pull/696/commits/7d82c760c632a4fdd7e9030bd7f51b88db621e9e